src/averageAges.js

```diff
@@ -14,12 +14,18 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = null) {
+  let menOnly = filterPersonSex(people, 'm');
+
+  if (century !== null) {
```

Comment:
I think it could be better to rewrite `if` to `ternary operator` to avoid nesting, but there is nothing wrong with this solution